types/boolean.pp

```diff
@@ -0,0 +1 @@
+type Accounts::Boolean = Variant[Boolean, Enum['no', 'yes']]
```

Comment:
What is the purpose of this type? This adds more complexity on the top of the Boolean type without really adding any value as far as I can tell.

Reply:
`yes` and `no` can be used with core Puppet Boolean type:
https://github.com/puppetlabs/puppet/blob/ba4d1a1aba0095d3c70b98fea5c67434a4876a61/spec/unit/functions/new_spec.rb#L507

Reply:
Removed.